openmetadata-ui/src/main/resources/ui/src/components/CustomEntityDetail/AddCustomField/RightPanel.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+
+export const RightPanel = () => {
+  return (
+    <Fragment>
+      <h6 className="tw-heading tw-text-base" data-testid="header">
+        Lorem, ipsum.
+      </h6>
+      <div className="tw-mb-5" data-testid="body">
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci ab,
+        rem molestiae culpa iure fugit cumque vel, nisi quam sed reprehenderit
+        exercitationem assumenda odio doloribus corrupti magni perferendis
+        veniam minus porro. Itaque accusamus veritatis esse laborum odit, nobis
+        sit quibusdam modi dolores iusto voluptas labore excepturi, tempora
+        praesentium nemo voluptatem?
```

Comment:
Can we add some meaningful information here?

Reply:
@shahsank3t, Sure. I will sync up with @ShilpaVernekar and replace this lorem lipsum with actual information
.